Conductor/Definition/TaskType/DynamicForkTask.cs

```diff
@@ -0,0 +1,12 @@
+using Conductor.Client.Models;
+
+namespace Conductor.Definition.TaskType
+{
+    public class DynamicForkTask : Task
+    {
+        public DynamicForkTask(string taskReferenceName, string taskNameParameter) : base(taskReferenceName, WorkflowTask.WorkflowTaskTypeEnum.DYNAMIC)
```

Comment:
taskName is not required and we should just default it to reference name.  This applies to all the tasks which are not simple tasks.

Reply:
I think it's not the task name of the task we're creating, but the simple task name to be triggered at this DynamicFork. Got it that we should have better name and also a few other constructor options 🤔